src/Native/Runtime/PalRedhawkCommon.h

```diff
@@ -134,6 +134,12 @@ struct PAL_LIMITED_CONTEXT
     UIntNative GetIp() const { return IP; }
     UIntNative GetSp() const { return Rsp; }
     UIntNative GetFp() const { return Rbp; }
+
+#ifdef UNIX_X86_ABI
```

Comment:
This ifdef is unnecessary.

Reply:
Removing this delta gives error on x86, adding SetIp and SetSp without ifdef statement will be unused on other architectures.

This is the error I received:

```sh
/corert/src/Native/Runtime/EHHelpers.cpp:401:15: error: no member named 'SetSp' in 'PAL_LIMITED_CONTEXT'
    pContext->SetSp(sp+sizeof(UIntNative)); // pop the stack
    ~~~~~~~~  ^
/corert/src/Native/Runtime/EHHelpers.cpp:445:21: error: no member named 'SetIp' in 'PAL_LIMITED_CONTEXT'
        palContext->SetIp((UIntNative)&RhpThrowHwEx);
        ~~~~~~~~~~  ^
```

Reply:
It is fine to have the functions there, but have them unused in some build configs.

Reply:
Removed in the commit fixup.